src/modules/email/domain/email-event-handler.ts

```diff
@@ -103,13 +133,17 @@ export class EmailEventHandler {
         this.logger.info(`Successfully deactivated email-address:${event.emailAddress}`);
     }
 
-    private async anyRolleReferencesEmailServiceProvider(rollen: Rolle<true>[]): Promise<boolean> {
-        const pro: Promise<boolean>[] = rollen.map((rolle: Rolle<true>) =>
+    private async anyRolleReferencesEmailServiceProvider(rollen: Rolle<true>[]): Promise<Option<string>> {
```

Comment:
anyRolleReferencesEmailServiceProvider -> findAnyRolleReferencesEmailServiceProvider oder getAnyRolleReferencesEmailServiceProvider

Reply:
heißt nun "getAnyRolleReferencesEmailServiceProvider", möchte das find wenn möglich eher in Repos nutzen